alvr/server_core/src/web_server.rs

```diff
@@ -61,14 +62,55 @@ pub async fn web_server(connection_context: Arc<ConnectionContext>) -> Result<()
     }
 
     let router = Router::new()
-        .route("/api/dashboard-request", routing::post(dashboard_request))
-        .route("/api/events", routing::get(events))
-        .route("/api/set-buttons", routing::post(set_buttons))
-        .route(
-            "/api/version",
-            routing::get(async || alvr_common::ALVR_VERSION.to_string()),
+        .nest(
+            "/api",
+            Router::new()
+                .route("/events", routing::get(events_websocket))
+                .route("/log", routing::post(set_log))
+                .nest(
+                    "/session",
+                    Router::new()
+                        .route("/", routing::get(get_session).post(update_session))
```

Comment:
I think it's probably better to have it as `/list` or `/get` and `/update` rather than overloading the endpoint (this is quite subjective).